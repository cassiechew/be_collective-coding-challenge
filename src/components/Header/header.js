import './header.scss';

/**
 * Component to hold a header
 * 
 * @Component
 */
export default function Header(props) {
    return (
        <div className="container__header">
            <h1>{props.title}</h1>
        </div>
    )
}