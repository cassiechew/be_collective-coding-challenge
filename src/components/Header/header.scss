div {
    .container__header {
        padding-left: 15%;
    }
}

h1 {
    margin-top: 0;
    padding-top: 5%;
}