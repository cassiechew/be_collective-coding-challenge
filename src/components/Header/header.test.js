import { render, screen } from '@testing-library/react';
import { create } from 'react-test-renderer';
import React from 'react';
import Header from './header';

describe('Header component', () => {
    test('contains header text', () => {
        render(<Header title="test" />);
        const text = screen.getByText(/test/i);
        expect(text).toBeInTheDocument();
    });
});

describe('snapshot test', () => {
    test('testing footer', () => {
      let tree = create(<Header title="test" />);
      expect(tree.toJSON()).toMatchSnapshot();
    });
});