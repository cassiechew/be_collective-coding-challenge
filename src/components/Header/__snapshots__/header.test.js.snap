// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test testing footer 1`] = `
<div
  className="container__header"
>
  <h1>
    test
  </h1>
</div>
`;
